backend/src/main/kotlin/app/ehrenamtskarte/backend/config/BackendConfiguration.kt

```diff
@@ -23,14 +23,16 @@ data class MapConfig(val baseUrl: String)
 data class GeocodingConfig(val enabled: Boolean, val host: String)
 data class CsvWriterConfig(val enabled: Boolean)
 data class SmtpConfig(val host: String, val port: Int, val username: String, val password: String)
+data class MatomoConfig(val siteId: Int, val url: String, val accessToken: String)
```

Comment:
Maybe we should do some sanity check that two different projects do not have the exact same matomo config?
Might prevent data corruption when new projects are added with copy pasta mistakes :D